TACC/TACC-Docs
# fix: headings as black text not purple links

## Overview

**Make headings black text not purple links.**
I.e. Rely on TOC permalinks (from #39) instead of manual ones.

## Related

- uses #39

## Changes

- **replaces** all `# [Something](#something)` with `# Something { #something }`

## Testing

1. Visit http://localhost:8000/software/pylauncher/#setup.
2. Verify "Basic Setup" heading is **not** purple and **not** a link.
3. Hover over "Basic Setup".
4. Verify "Basic Setup" has a working link icon to the left of it.

## UI

| before | after |
| - | - |
| <img width="1194" alt="before" src="https://github.com/user-attachments/assets/5cd0a064-d8f3-4b88-a1c1-64bc13a3c3ed"> | <img width="1194" alt="after" src="https://github.com/user-attachments/assets/5547e887-d084-464f-9e9c-9e24d1f3a00b"> |

https://github.com/user-attachments/assets/033edd86-2a50-4848-b055-e9144a8838a6
